dotcom-rendering/src/web/server/index.ts

```diff
@@ -170,3 +171,25 @@ export const renderKeyEvents = (
 		res.status(500).send(`<pre>${message}</pre>`);
 	}
 };
+
+export const renderFront = (
+	{ body, query }: express.Request,
+	res: express.Response,
+): void => {
+	try {
+		const slug: string =
+			query?.url
+				?.toString()
+				.replace('https://www.theguardian.com/', '/') ??
+			'Missing url query param';
```

Comment:
Just checking: The intention is to remove this code later?

Reply:
I’ve kept all the dummy code inside `frontToHtml` now.